cardigan/stories/components/BackToResults.js

```diff
@@ -3,23 +3,23 @@ import BackToResults from '../../../common/views/components/BackToResults/BackTo
 import Readme from '../../../common/views/components/BackToResults/README.md';
 
 const nextLink = {
-  'href': {
-    'pathname': '/works',
-    'query': {
-      'query': 'sun',
-      'page': 2
-    }
+  href: {
+    pathname: '/works',
+    query: {
+      query: 'sun',
+      page: 2,
+    },
+  },
+  as: {
+    pathname: '/works',
+    query: {
+      query: 'sun',
+      page: 2,
+    },
   },
-  'as': {
-    'pathname': '/works',
-    'query': {
-      'query': 'sun',
-      'page': 2
-    }
-  }
 };
 
 const stories = storiesOf('Components', module);
-stories.add('BackToResults', () => (
-  <BackToResults nextLink={nextLink}  />
-), {info: Readme});
+stories.add('BackToResults', () => <BackToResults nextLink={nextLink} />, {
```

Comment:
Why not use `displayName`?
Non fussed, but it does stop typos etc.

Reply:
That's a good shout

Reply:
`BackToResults.displayName` is `undefined` – what am I missing?

Reply:
Hmm. Tried the same thing, might do it as a second step.